src/Tribe/Views/V2/Behaviors/Breakpoint_Behavior.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Provides methods for Views for breakpoint behavior.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Behaviors
+ */
+
+namespace Tribe\Events\Views\V2\Behaviors;
```

Comment:
Question: could you explain why you've created a `Behaviors` directory for this? Do you think we'll add more?

Reply:
Yeah, I had a convo with Paul around possibly moving some things out of the View Class, since currently, it creates a lot of bloat. Tomorrow lets talk about that, I think it could be a possible solution. Mostly around we could use for Breadcrumbs, Messages, and move the currently existing `List_Behavior` into that folder.